test/cql/cdc_disallow_cdc_for_counters_test.result

```diff
@@ -0,0 +1,25 @@
+create table tb1 (pk int primary key, c1 counter) with cdc = {'enabled': true};
+{
+	"message" : "exceptions::invalid_request_exception (Cannot create CDC log for a table ks.tb1, because CDC for counters is not implemented)",
```

Comment:
I don't really like testing for exact messages. Could we have some less volatile way to communicate the error type/cause?

Reply:
I don't think cql tests allow anything less specific, do they? If not then I don't think this PR is a place to add sth like that.

Reply:
Neither do I know of a way to do it in CQL tests. But comparing the exact error messages was being done for a long time, so it's "our standard practice".